### ISSUE TITLE
ultra-short, illustrative story for patterns

### ISSUE THREAD

**Issue by fwan2000** [InnerSourceCommons/InnerSourcePatterns#152]
Story provides a more narrative, less academic description to a pattern. When a pattern has a good narrative description at the first few sections, it helps people deciding quickly whether this is something they want to dive deeper. Some patterns provide good narrative while some other patterns lack those narrative angle. It would help to add story or improve narrative description to the patterns that don't have good narratives.

**Comment by NewMexicoKid** [comment]
We've used case studies in the past for illustrative purposes, thinking that actual cases would accurately show the context for the pattern. Are you thinking of an artificial story just for illustrative purposes?

**Comment by fwan2000** [comment]
Yes I'm thinking of short story for illustrative purposes. Some patterns have them already. I find them helpful to understand the pattern at high level before diving into the details.

**Comment by maxcapraro** [comment]
I really like the way Mary Lynn and Linda did it in [Fearless Change](https://www.amazon.de/Fearless-Change-Patterns-Introducing-Ideas/dp/0134395255). Their book has two parts:

- Chapters delivering a story or experience report and referencing related patterns (-> making for an easy and fun reading experience) 
- A full list of all patterns (-> making it easy to go back and find that one specific pattern a reader needs)

I feel this could be a good format for our e-book. This delivers on the story telling ancle that you proposed, @fwan2000.

**Comment by maxcapraro** [comment]
(This is a hybrid between "Content Work" and "Meta" ... I marked it as `Type - Meta` because it proposes a change to one of the process documents / conventions (patterns format file).

**Comment by spier** [comment]
@fwan2000 this is a great idea. 


Right now this issue talks about a general idea for all patterns. 

Some ideas to make this more executable:
- which of the existing patterns already have the type of **Story** (narrative) that you are looking for?
- which of the existing patterns lack the same.
- could you provide an example of such a story for one such pattern?

**Comment by fwan2000** [comment]
A quick grep in structured directory shows the following patterns having a story:

dedicated-community-leader.md:## Story
gig-marketplace.md:## Story
innersource-portal.md:## Story
repository-activity-score.md:## Story
transparent-cross-team-decision-making-using-rfcs.md:## Story

We may not need to have a story for every single pattern. It may worth adding a story for the upcoming pattern that will have its deep dive.

**Comment by spier** [comment]
Good idea to grep for this :)

Do you mean "deep dive" as in "pattern of the month" #280?

This thread certainly got me thinking. No coherent thoughts yet, so I am just listing them here to see if any of them connect.

### "Quickly" (speed of reading)

There is an interesting issue here:

* The **Patlet** is meant to tell readers at a glance "does this sound relevant/interesting for me at all" - I would expect this to be very short, so that I can read many of them quickly
* Then the **Problem** plus sections like **Context/Forces** help to find out "do I have a similar situation, i.e. is the **Solution** applicable to me".

Now the **Story** part could be filled with more narrative, as you are saying. This would likely not be using any bullet points, as it should feel like a story that somebody is telling you, right? Also by the nature of it being a story it will be more text, certainly more text than a Patlet.

So reading many stories would take more time, and with that may beat the purpose you were after initially?
> ... it helps people deciding **quickly** whether this is something they want to dive deeper

_Quickly_ is quite a relative term of course :)

### Narration / Chatham / Pattern-style

Has the Chatham House Rule lead authors to try to remove company-specific info from the Pattern? Or maybe that has happened to make the pattern more generally applicable and translatable to another company?

Certainly if that was true, it would make it harder to share good stories, because stories tend to be "personal", and about a particular organization. To tell it well, one needs to be specific. Or not?

### Stories => real book

Stories would certainly create great material for a real book! The [book we we have now](https://innersourcecommons.gitbook.io/innersource-patterns/) we have right now is really mostly a **catalog** of patterns, as it doesn't have much material that can serve as a red thread to tie it all together.

Stories might help there. i.e. one could even just publish the stories, and then from there link to the patterns in the catalog.
